lib/cs_guide_web/controllers/brand_controller.ex

```diff
@@ -152,47 +152,6 @@ defmodule CsGuideWeb.BrandController do
     |> Enum.take(4)
   end
 
-  @doc """
-  Returns the most common used drink type of a brand
-  This is used to define the background colour of the brand page
-  see issue #346 and the PR #384
-  """
-  def get_drink_type(brand) do
-    brand.drinks
-    |> Enum.flat_map(fn drink ->
-      Enum.map(drink.drink_types, fn type -> type.name end)
-    end)
-    |> max_by_name()
-  end
-
-  @doc """
-  Returns the most common drink's style used by the brand
-  see issue #598
-  """
-  def get_drink_style(brand) do
-    brand.drinks
-    |> Enum.flat_map(fn drink ->
-      Enum.map(drink.drink_styles, fn style -> style.name end)
-    end)
-    |> max_by_name()
-  end
-
-  @doc """
-  Returns the most common element of a list
-  ## Example
-      iex> max_by_name(["one", "two", "one"])
-      "one
-  """
-  def max_by_name([]), do: nil
```

Comment:
Move these function to `Resources.Drink` module to be able to reuse them in venue controller